sample_client/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--  ownCloud Android Library is available under MIT license
+<?xml version="1.0" encoding="utf-8"?><!--  ownCloud Android Library is available under MIT license
```

Comment:
is there any strong reason to change this?

Reply:
No